dashboard/test/controllers/api/v1/pd/workshop_survey_foorm_report_controller_test.rb

```diff
@@ -225,6 +251,35 @@ class WorkshopSurveyFoormReportControllerTest < ::ActionController::TestCase
       assert_nil response[:workshop_rollups][:facilitator][:single_workshop][facilitator_2_id]
     end
 
+    test 'participant and facilitator post surveys are available for download' do
+      csf_workshop = create :csf_workshop,
+                            started_at:  Time.now.utc - 1.day,
+                            ended_at: Time.now.utc - 1.hour,
+                            num_facilitators: 2
+      facilitator = csf_workshop.facilitators[0]
+      csf_workshop.facilitators[1]
+      create_surveys_for_csf_workshop(csf_workshop, csf_workshop.facilitators.pluck(:id), 5, 2)
+      create :facilitator_post_survey_workshop_submission, user: facilitator, pd_workshop: csf_workshop
+
+      get :forms_for_workshop, params: {workshop_id: csf_workshop.id}
+      assert_response :success
+
+      response = JSON.parse(@response.body)
+
+      form_names = [
+        'surveys/pd/workshop_csf_intro_post_test',
+        'surveys/pd/csd_csp_facilitator_post_survey'
+      ]
+      assert_equal form_names, response.map {|h| h['name']}
+
+      sign_in @workshop_admin
```

Comment:
What kinds of permissions are required to see these surveys? I assumed facilitators would see the survey results for participants, but they wouldn't see other facilitator results. Not sure though. I see this logic
https://github.com/code-dot-org/code-dot-org/blob/73e404a4240f57d5a88585892a10999996277d01/dashboard/app/controllers/api/v1/pd/workshop_survey_foorm_report_controller.rb#L53-L55

but not sure what's expected for surveys submitted by facilitators

Reply:
yep, good question! there are other test cases in this file which validate what facilitators can see if they look at this page: https://github.com/code-dot-org/code-dot-org/blob/9049f58433c01a75c622aaf4ea8831dd89009da5/dashboard/test/controllers/api/v1/pd/workshop_survey_foorm_report_controller_test.rb#L218

Reply:
Ah I see! Thanks